src/leader.py

```diff
@@ -58,10 +58,12 @@ def set_speed(self, position):
         :param position: The value of the button input.
         :type position: int
         """
+        percent_top_speed = (position + 1)/2
+
         if position == -1:
             speed = 0
         else:
-            speed = self.MAX_SPEED
+            speed = self.MAX_SPEED * percent_top_speed
```

Comment:
Get rid of the "self" here. That's my fault. Somehow slipped through.

Reply:
You need to cast this to an int because the SunFounder code freaks out if it's a float. But basically I want this line to be 
```python
speed = int(MAX_SPEED * ((position + 1) / 2))
```